### PR TITLE
First changes for pi-gen screenly

### PR DESCRIPTION
**Pull request by rusko124**
PR for https://github.com/Screenly/screenly-ose/issues/711

### CODE REVIEW

**Comment by vpetersson** [comment]
Looks good, @rusko124. 

Two questions:
 * How big is this disk image compared to the base line?
 * Did you boot it and confirm that the generated file works?

File: stage2/03-screenly/01-run.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+# Add checkout to production branch later
+
+on_chroot << EOF
+  git clone https://github.com/Screenly/screenly-ose.git /home/pi/screenly
```
Comment (vpetersson) [inline stage2/03-screenly/01-run.sh:6]: This would install the development branch (master) instead of the production version.

**Comment by rusko124** [comment]
> Looks good, @rusko124.
> 
> Two questions:
> 
> * How big is this disk image compared to the base line?
> * Did you boot it and confirm that the generated file works?

1)2.7GB now, last image release 4GB (i think we can try make the image more lighter).
2) Yes, I was able to boot and run. (But need more tests and deep check logs.)

**Comment by vpetersson** [comment]
@rusko124 can you re-issue the pull request with the fix above? 

Also, can you verify that no swap space is created?

**Comment by rusko124** [comment]
@vpetersson 
https://github.com/Screenly/pi-gen/pull/3/commits/e7bf04001c1c805be324584814b393e45a84481c
These are the latest changes with which I successfully created the image.
System: Debian 9
Image size: 2.2GB

Suggestions to make image lighter:
- After ansible work, we can delete all dependencies.
- Also we can delete packages from https://github.com/rusko124/pi-gen/blob/master/stage2/01-sys-tweaks/00-packages

Unfortunately i couldn't to solve the error with the Docker, and to create the image there, it is necessary to restart after the error with the CONTINUE=1.

**Pull request merged**